### ISSUE TITLE
How to check per image inference time for any custom object detection model?

### ISSUE THREAD

**Issue by minhaz109074** [tensorflow/models#10494]
Hello people,
I trained SSD-MobileNetv1 on my custom dataset . Now I want check the time it take to detect a particular image on gpu and cpu.
Please help me to do this task.

**Comment by Kannan665** [comment]
You can use the following python codes to check for inferencing using your system's CPU or GPU.......I referred to this github page and made few changes to the code, to check for inferencing on my saved_model on images and videos. While inferencing the videos, the frame rate is displayed which is one of the metric to be observed, if you want your model to do liver inferencing........https://github.com/abdelrahman-gaber/tf2-object-detection-api-tutorial.....While you go through the link above, will try to get the arguments to be passed for doing the inferencing using the saved_model on your CPU/GPU......
[detect_objects.zip](https://github.com/tensorflow/models/files/8061113/detect_objects.zip)

**Comment by minhaz109074** [comment]
Thanks for your reply . I just completed inferencing. I faced some issues while running the script  you have given in Colab. But I solved these issues. Now can you please tell me how to find out flops/params of the saved model.

**Comment by kumariko** [comment]
[minhaz109074](https://github.com/minhaz109074) Could you please refer to this [link](https://stackoverflow.com/questions/45085938/tensorflow-is-there-a-way-to-measure-flops-for-a-model) to know about flops of the models and let us know if it helps. Thanks!

**Comment by minhaz109074** [comment]
@[kumariko](https://github.com/kumariko) The link you provided contains code for keras sequential model. But I have a  saved model in .pb format. Is there any code snippet which takes .pb format model as arguments  and returns flops/params?

**Comment by Annieliaquat** [comment]
> @[Kannan665](https://github.com/Kannan665) Thanks for your reply . I just completed inferencing. I faced some issues while running the script you have given in Colab. But I solved these issues. Now could you please tell me how to find out flops/params of the saved model.

Can you please let me know that how did you find the inference time??
